person.rb

```diff
@@ -0,0 +1,20 @@
+# Person
+class Person
+  attr_accessor :name, :age, :id
+  attr_reader :parent_permission
```

Comment:
- [ ] Kindly use `attr_accessor` for `name` and `age` and `attr_reader` for `id`